diff --git a/server/intel/i915/i915_drm.cpp b/server/intel/i915/i915_drm.cpp
index 3ca0f52..7bf7c58 100644
--- a/server/intel/i915/i915_drm.cpp
+++ b/server/intel/i915/i915_drm.cpp
@@ -1,6 +1,6 @@
 #include <stdio.h>
 #include <stdint.h>
-#include <drm/i915_drm.h>
+#include <libdrm/i915_drm.h>
 #include <errno.h>
 #include <stdlib.h>
 #include <assert.h>
diff --git a/server/intel/xe/xe_drm.cpp b/server/intel/xe/xe_drm.cpp
index 85d8d0b..d20eaf2 100644
--- a/server/intel/xe/xe_drm.cpp
+++ b/server/intel/xe/xe_drm.cpp
@@ -1,6 +1,6 @@
 #include <stdio.h>
 #include <stdint.h>
-#include <drm/xe_drm.h>
+#include <libdrm/xe_drm.h>
 #include <errno.h>
 #include <stdlib.h>
 #include <assert.h>
